Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -743,3 +743,38 @@ This capability is not supported on Windows.
 (3) Memory profiling is an optional feature in AMReX that is not enabled by default.  It reads
 memory system information from the OS to give us a summary of our memory usage.  This is not
 supported on Windows.
+
+
+.. _sec:build:spack:
+
+Spack
+=====
+
+AMReX can be installed using the scientific software package manager Spack. Spack is designed
```

Comment:
Maybe it's more appropriate to say `Spack supports ...` instead.

Reply:
Yes, that is better.